### ISSUE TITLE
Error with 0 decimals

### ISSUE THREAD

**Issue by Calvein** [bevry/cson#56]
I'm not sure how to actually call them but I'm talking about number like this `0.4`.
We have to put a `0` before the `.` or it returns an empty object (in the cli).

``` coffeescript
# in.cson
test: 0.2
# out.json
{
  "test": 0.2
}
```

``` coffeescript
# in.cson
test: .2
# out.json
{}
```

**Comment by pflannery** [comment]
It seems JSON.parse has a similar scenario expect it doesn't return ```{}```

```js
JSON.parse('{"test": .1}') // Gives (program):1 Uncaught SyntaxError: Unexpected token .

JSON.parse('{"test": 0.1}') // Gives Object {test: 0.1}
```

**Comment by balupton** [comment]
Probably due to https://github.com/bevry/cson/issues/78

**Comment by balupton** [comment]
closed with v4.1.0
